jmp-cloud-service-impl/src/main/java/com/gmail/iikaliada/impl/SubscriptionConverterImpl.java

```diff
@@ -0,0 +1,25 @@
+package com.gmail.iikaliada.impl;
+
+import com.gmail.iikaliada.converter.SubscriptionConverter;
+import com.gmail.iikaliada.dto.SubscriptionRequestDto;
+import com.gmail.iikaliada.dto.SubscriptionResponseDto;
+import com.gmail.iikaliada.entity.Subscription;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SubscriptionConverterImpl implements SubscriptionConverter {
+
+    private final ModelMapper modelMapper = new ModelMapper();
+
+    @Override
+    public SubscriptionResponseDto convertSubscriptionToSubscriptionDto(Subscription subscription) {
```

Comment:
Minor: 

Method name could be simplified smth like `toSubscriptionDto`. There is no sense in **convert** verb here as the class name suggests that this object should convert something